src/server/mpsserver.ts

```diff
@@ -403,10 +403,10 @@ export class MPSServer {
       case APFProtocol.CHANNEL_CLOSE: {
         if (len < 5) return 0
         const RecipientChannel: number = common.ReadInt(data, 1)
-        this.debug(3, 'MPS:CHANNEL_CLOSE', RecipientChannel.toString())
+        log.silly(`MPS:CHANNEL_CLOSE ${RecipientChannel.toString()}`)
         const cirachannel = socket.tag.channels[RecipientChannel]
         if (cirachannel == null) {
-          log.debug(`MPS Error in CHANNEL_CLOSE: Unable to find channelid ${RecipientChannel}`); return 5
+          log.warn(`MPS Error in CHANNEL_CLOSE: Unable to find channelid ${RecipientChannel}`); return 5
```

Comment:
I dont know about warn for these, its a bit confusing that the message says Error but the level is warn. Thoughts?

Reply:
i'm ok with changing this one and any other similar ones to error if you like?

Reply:
i think yeah change it.